Makefile

```diff
@@ -174,3 +176,101 @@ build-snap:
 
 install-snap:
 	@sudo $(SNAP) install axolotl_$(AXOLOTL_VERSION)_amd64.snap --dangerous
+
+## Debian arm64 building and packaging
+## Please get the source via
+##  go get -d -u github.com/nanu-c/axolotl/
+check-platform-deb-arm64:
+	@echo "Building Axolotl for Debian arm64/aarch64"
+  ifneq ($(shell uname),Linux)
```

Comment:
On other places, for example line 131, the following is used:

```
ifeq ($(UNAME_S), Linux)
```

Does that work here as well?

Reply:
I can use your variable UNAME_S, no problem!